build_targets/families.py

```diff
@@ -175,10 +175,7 @@ def ld_flags(self) -> List[str]:
         ]
 
     def dependencies(self) -> List[BuildDependency]:
-        cmsis_deps = [
+        return [
             CMSISDependency(components=['Core']),
             CMSISDeviceEFM32GG12BDependency(),
         ]
```

Comment:
You removed the tinyusb dep here?

Reply:
leftovers from debugging the build system, ups